portfolio/src/main/webapp/change_picture_script.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Generates a random number i from 1 to 16. Then create the URL that points to
+ * the ith landscape image in the images folder and add the image element to the
+ * container of the gallery page.
+ */
+function changeImage() {
+    // Generate a random integer from 1 to 16
+    const index = Math.floor(Math.random() * 16) + 1;
+    // Create the URL that points to the certain landscape image in the images folder
+    const imageUrl = "images/landscape-" + index + ".jpg";
+
+    const newImgElement = document.createElement("img");
```

Comment:
Can you simply update the `src` attribute of the existing `img` element, rather than completely replacing the element each time?

Reply:
I just deleted this part of script and replaced it with my new sliding window feature.